clients/drcachesim/tests/offline-burst_threadfilter.templatex

```diff
@@ -21,7 +21,7 @@ Total counts:
     *[0-9]* total data stores
     *[0-9]* total icache flushes
     *[0-9]* total dcache flushes
-           4 total threads
```

Comment:
Since we have your `burst_replaceall.cpp` fix to make threads wait for attach to complete, won't we surely see all threads? Why did we need to relax this?

Reply:
This is a different test, burst_threadfilter.cpp.  It is more complex and looked harder to add the same type of synchronization so I went with the relax-template approach.

Reply:
Ah, missed that simple fact. Thanks.